backend/requirements.txt

```diff
@@ -34,34 +41,52 @@ langchain-text-splitters==0.3.0
 langgraph==0.2.34
 langgraph-checkpoint==2.0.0
```

Comment:
Please consider adding a check for the new dependencies in the code patch, especially markdown-it-py==3.0.0 and pydub==0.25.1 which were added recently.